avaframe/tests/data/testCom1DFA/getIniP_com1DFACfg.ini

```diff
@@ -0,0 +1,273 @@
+### Config File - This file contains the main settings for the simulation run
+## Copy to  local_com1DFACfg.ini and set you parameters
+# This file is part of Avaframe.
+
+
+[GENERAL]
+#++++++++++++++++ Simulation type
+# list of simulations that shall be performed (null, ent, res, entres, available (use all available input data))
+simTypeList = null
+# model type - only for file naming (dfa - dense flow avalanche)
+modelType = dfa
+
+#+++++++++++++ Output++++++++++++
+# desired result Parameters (ppr, pfd, pfv, FD, FV, P, Vx, Vy, Vz, particles) - separated by |
+resType = ppr|pfd|pfv
+# saving time step, i.e.  time in seconds (first and last time step are always saved)
+# option 1: give an interval with start:interval in seconds (tStep = 0:5 - this will save desired results every 5 seconds for the full simulation)
+# option 2: explicitly list all desired time steps (closest to actual computational time step) separated by | (example tSteps = 1|50.2|100)
+# NOTE: initial and last time step are always saved!
```

Comment:
I would just put the parameters that you changed here to make it a bit clearer

Reply:
changed